symforce/test_util/backend_coverage_expressions.py

```diff
@@ -11,7 +11,7 @@
 from symforce import typing as T
 
 
-def backend_test_function(x: sf.Scalar, y: sf.Scalar) -> T.Tuple[sf.Scalar, ...]:
+def backend_test_function(non_standard_unary_ops, x: sf.Scalar, y: sf.Scalar) -> T.Tuple[sf.Scalar, ...]:
```

Comment:
I did this to preserve functionality in tests for cuda, but not have to implement things like gamma loggamma for rust which are only available on nightly.

Reply:
Yeah, I think this makes sense